src/api/tasks/__tests__/TaskCollaborators.test.js

```diff
@@ -75,6 +78,40 @@ describe('api/TaskCollaborators', () => {
             });
         });
 
+        describe('createTaskCollaboratorsforGroup()', () => {
```

Comment:
Adding a test for `createTaskCollaboratorsforGroup` as it was missed in a previous PR.